.design/experiments/r3/spacing-bench.html

```
@@ -0,0 +1,343 @@
+<title>Resources Header Lab</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)